src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotifyClaimDetailsCallbackHandler.java

```diff
@@ -138,6 +150,33 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
             .build();
     }
 
+    private CaseData prepareCoSDetails(CaseData caseData, int lipNumber) {
```

Comment:
I would suggest to method name like "saveDocument...." or something. As "prepareCosDetails" doesnt tell what this method is exaclty doing.